### ISSUE TITLE
Example doesn't seem to work properly.

### ISSUE THREAD

**Issue by NoSmoke999** [arduino-libraries/Arduino_DebugUtils#8]
The example:
`
int i = 1;
float pi = 3.1459;
Debug.print(DBG_VERBOSE, "i = %d, pi = %f", i, pi);`

Results in:
 i = 1, pi = ?

It looks like  pi=%f  doesn't work.

**Comment by per1234** [comment]
Hi @NoSmoke999. I'm guessing you're using one of the AVR boards like the Uno, classic Nano, Pro Mini, Leonardo, Micro, or Mega. Is that correct?

If so, I think the situation is discussed here: https://github.com/arduino/ArduinoCore-avr/issues/293

**Comment by NoSmoke999** [comment]
Hey, correct - using a Uno.  The discussion in the link you provided is way above my pay grade so I'll just assume for now that floating pt print can't be done now on an AVR board(?).  Anyhow, thank you for the info. :)

**Comment by klodoma** [comment]
It's ok not to work and it makes sense. 
It should be made clear in the library description for which Arduino boards it works, not to use it on Uno.

**Comment by klodoma** [comment]
This seems to be a good way to output floats on `nano` 
```c++
float temp=1.2345;
Debug.print(DBG_INFO, "Temp: %s", String(temp, 2).c_str());
```

**Comment by aentinger** [comment]
Closed due to #26 .
